app/views/content/shared/age-groups-and-specialisms/_ages_abilities.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="teaching-pupils">
```

Comment:
This is a direct copy of the partial used on the subject page (apart from the QTS link) `https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/content/shared/teaching/_ages_abilities.html.erb`

I suspect you have created a duplicate partial because you link to QTS further up the secondary school page?

However, I'm not sure that this is the right approach - we are then having to maintain two partials, which is not really the point of them!

I would suggest using the existing partial and perhaps removing the earlier QTS link on the secondary page if necessary (or keeping both if absolutely necessary)

But do discuss with @Sarah-DfE :-) 

Either way, this partial should not be in an `age-groups-and-specialisms` folder - folders should relate to what the partial is about rather than where it is being used!